### PR TITLE
refine/add some logs

### PR DESCRIPTION
**Pull request by SeaRise**
### What problem does this PR solve?

Issue Number: ref #5900 

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Flash/Mpp/MPPTask.cpp [hunk 1]
```diff
@@ -324,7 +324,9 @@ void MPPTask::preprocess()
 {
     auto start_time = Clock::now();
     initExchangeReceivers();
+    LOG_DEBUG(log, "init exchange receiver");
```
Comment (windtalker) [inline dbms/src/Flash/Mpp/MPPTask.cpp:327]: `init exchange receiver done`?

File: dbms/src/Flash/Mpp/MPPTask.cpp [hunk 2]
```diff
@@ -324,7 +324,9 @@ void MPPTask::preprocess()
...
     query_executor_holder.set(queryExecute(*context));
+    LOG_DEBUG(log, "init query executor");
```
Comment (windtalker) [inline dbms/src/Flash/Mpp/MPPTask.cpp:329]: `init query executor done`?

Comment (SeaRise) [reply dbms/src/Flash/Mpp/MPPTask.cpp]: updated.

**Review by windtalker — APPROVED** [review]
LGTM

**Review by Lloyd-Pottiger — APPROVED** [review]

**Review by LittleFall — APPROVED** [review]

**Pull request merged**